src/managed/dir.proj

```diff
@@ -16,6 +16,7 @@
       <BuildArgs>/p:Configuration=$(ConfigurationGroup)</BuildArgs>
       <BuildArgs Condition="'$(LatestCommit)' != ''">$(BuildArgs) /p:LatestCommit=$(LatestCommit)</BuildArgs>
       <BuildArgs Condition="'$(BuiltByString)' != ''">$(BuildArgs) /p:BuiltByString="$(BuiltByString)"</BuildArgs>
+      <BuildArgs>$(BuildArgs) /p:BaseOutputPath=$(BaseOutputRootPath)</BuildArgs>
```

Comment:
`BaseOutputPath` is used by the sdk to determine where to put binary output - https://github.com/dotnet/sdk/blob/c6542db9e04b376d8a2f38574bd6eb3ea36b78e5/src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.DefaultOutputPaths.targets#L33-L43

Microbuild will only sign binaries which are under either the `OutDir` folder, or `IntermediateOutputPath` https://github.com/dotnet/sdk/blob/c6542db9e04b376d8a2f38574bd6eb3ea36b78e5/src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.DefaultOutputPaths.targets#L33-L43

So, we need the PlatformAbstractions and DependencyModel binaries to be produced under `BaseOutputRootPath` so that they can be signed, and then from there, picked up for packaging.